package.json

```diff
@@ -78,14 +80,15 @@
     ]
   },
   "scripts": {
-    "cleanup": "rimraf dist/*",
+    "cleanup": "cross-var rimraf $npm_package_ngPackage_dest/*",
```

Comment:
Do it work with `yarn`?

Reply:
Yes, this is working with `yarn`, but anyway I try to remove this.